SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -325,5 +325,5 @@ void USpatialWorkerConnection::CacheWorkerAttributes()
 	for (uint32 Index = 0; Index < Attributes->attribute_count; ++Index)
 	{
 		CachedWorkerAttributes.Add(UTF8_TO_TCHAR(Attributes->attributes[Index]));
-	}	
+	}
```

Comment:
@danielimprobable

Reply:
WHYYYYYYYYYYYYYYYYYYY, *in tears** #Sorry NOOOOOOOO